redaxo/src/core/lib/login/login.php

```diff
@@ -266,6 +266,7 @@ public function checkLogin()
                 if (1 == $this->user->getRows() && self::passwordVerify($this->userPassword, $this->user->getValue($this->passwordColumn), true)) {
                     $ok = true;
                     self::regenerateSessionId();
```

Comment:
TODO: `regenerateSessionId()` integrieren.

Reply:
es scheint nicht mehr notwendig, nachdem es in `rex_backend_login` verschoben wurde und nach erfolgreichem Login ausgeführt wird.